.eslintignore

```diff
@@ -0,0 +1,9 @@
+# Never lint node_modules
```

Comment:
We can remove this file as we're using the `ignorePatterns` section on `.eslintrc.js`